lib/org/chromium/webidl/WebKitParser.js

```diff
@@ -33,6 +33,9 @@ foam.CLASS({
                       repeat0(alt(notChars('*'), seq('*', notChars('\/')))),
                       '*\/'),
                   // C-style preprocessor directives.
+                  seq('#ifndef',
+                    repeat0(notChars('#')),
+                    '#endif'),
```

Comment:
The current code only recognized one line C-style preprocessor directives. Adding support for `#ifndef` here as it shows up quite often. May be changed in the future to account for other multiline directives